doc/configuration.rst

```diff
@@ -24,6 +24,10 @@ You can access ``some_value`` through the ``config`` project attribute:
 
 .. note:: Some keys in the project config have special meaning, like the project name stored in the ``name`` key or ``flask_config`` described below. To avoid name clashes, it is wise to store all of your specific configuration under a specific key as a nested structure, such as ``config`` or ``project-config``
 
+Cob Config Options
+------------------
+pypi_index_url - For use with pypi other than https://pypi.org/simple
+
 
```

Comment:
Add a section about `specific_virtualenv_pkgs`, the meaning of `COB_USE_PRE` , `COB_VERSION` and `COB_DEVELOP`